sass/_programs.scss

```diff
@@ -0,0 +1,65 @@
+@import "./variables";
+
+.main-program {
+  padding-top: 80px;
+  padding-bottom: 80px;
+  background-color: map-get($map: $colors, $key: "primary");
+  background-image:
+    radial-gradient(
+      rgb(48, 48, 48) 6px,
+      transparent 6px
+    );
+  background-size: 30px 30px;
+
+  @include flex-center(column);
+
+  .main-program__title {
+    color: #fff;
+    text-align: center;
+    font-size: 18px;
+  }
+
+  .main-program__line {
+    display: block;
+    margin-top: 15px;
+    background-color: map-get($map: $colors, $key: "secondary");
+    height: 2px;
+    width: 10%;
+  }
+
+  .main-program__container {
+    margin: 20px 0 20px 0;
+    gap: 7px;
+
+    .main-program__list {
+      width: 100%;
+      background-color: rgba(170, 167, 167, 0.25);
+      padding: 5px;
+
+      .list-head {
+        gap: 10px;
+
+        h3 {
+          color: map-get($map: $colors, $key: "secondary");
+          font-weight: map-get($map: $font_weights, $key: "bold");
+        }
+      }
+
+      p {
+        color: #fff;
+        font-size: 12px;
+      }
+
+      &:hover {
+        background-color: rgba(170, 167, 167, 0.479);
```

Comment:
- [ ] One more thing, can we please kindly design this according to the given template? The hover effect should place a border (color of #fff) around the selected card, I think that's what the previous reviewer meant by "whiteness around the cards".

## Given template
<img width="360" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/233301198-322c2df0-2fd5-43c8-ae1c-2363b6b46d3d.png">